08-components-lab/src/01-timeconverter.jsx

```diff
@@ -1,12 +1,47 @@
-/**
- * Lab 01: 
- *
- * Implement a time converter component.
- * Component should show 3 input fields labeled hours, minutes and seconds.
- * Typing a value in either field updates the other two with corresponding values.
- */
-
-import React from 'react';
-
-export default React.createClass({
-});
+import React from 'react';
+
+export default class TimeConverter extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {h: 0
+                     ,m:0

```

Comment:
you don't really need hours and minutes in state since they are derived from seconds.